app/src/app/[lng]/[inventory]/preferences/activities/ActivitiesPage.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { Heading, SimpleGrid, Text, VStack } from "@chakra-ui/react";
+import type { TFunction } from "i18next";
+import { PreferenceCard } from "@/app/[lng]/[inventory]/preferences/activities/PreferenceCard";
+import { ACTIVITY_ITEMS } from "@/app/[lng]/[inventory]/preferences/activities/ACTIVITY_ITEMS";
+
+export default function ActivitiesPage({ t }: { t: TFunction }) {
+  function onClick() {}
```

Comment:
What is this function for? Could we just remove the onClick property here and make it optional in PreferenceCard?

Reply:
the function will be used once we add the backend logic